src/main/frontend/components/block.cljs

```diff
@@ -2585,7 +2582,7 @@
         config (if (nil? (:query-result config))
                  (assoc config :query-result (atom nil))
                  config)
-        heading? (or (= type :heading) (and heading-level (<= heading-level 6)))
+        heading? (or (:heading properties) (and heading-level (<= heading-level 6)))
```

Comment:
Was this a necessary change? I don't know if heading-level goes with a :heading property. We don't have tests around this but it appears to go with `:type :heading` per https://github.com/logseq/logseq/blob/master/deps/graph-parser/src/logseq/graph_parser/block.cljs#L570-L573 /cc @tiensonqin

Reply:
I think it's a necessary change. Because headings include:
1. it has a `heading-level` property.
2. it has a `heading` property, and its value is true. (This is created by `Converting to heading`)